android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue Sep 27 21:00:27 EEST 2022
+distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionPath=wrapper/dists
+zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME

```

Comment:
Windows line endings?

Reply:
copied it from the repo as is. Building from Windows is also possible. Can change, probably Gradle doesn't care about line endings.

Reply:
Probably better to change. It is git who normally handles line endings, not developer (autoCLRF option or what its name).